pipelines/main/platforms/upload_macos.yml

```
@@ -0,0 +1,125 @@
+steps:
+  - label: ":macos: upload ${TRIPLET?}"
+    key:   "upload_${TRIPLET?}"
+    # We only upload to S3 if the branch is `master` or `release-*`.
+    if: ((build.branch == "master") || (build.branch =~ /^release-/))
+    depends_on:
+      # Wait for the builder to finish
+      - "build_${TRIPLET?}"
+      # Wait for the tester to finish
+      - "test_${TRIPLET?}"
+    # Prevent multiple pipelines from uploading to S3 simultaneously
+    # It is okay for two different triplets to upload simultaneously
+    concurrency: 1
+    concurrency_group: 'upload/julialangnightlies/upload_macos/${TRIPLET?}'
+    plugins:
+      - JuliaCI/external-buildkite#v1:
+              version: "./.buildkite-external-version"
+              repo_url: "https://github.com/JuliaCI/julia-buildkite"
+      - staticfloat/cryptic#v2:
+          variables:
+            - AWS_ACCESS_KEY_ID="U2FsdGVkX184v87+NPs3j9r/JoIuOrYt4/Z4wnRdklnY17NP8C8AMZvWYLJfT9t1"
+            - AWS_SECRET_ACCESS_KEY="U2FsdGVkX1+qptnxR/Mo5jZdH8OQfflRPiQBEhjgZIiTpn8KNCJYh/Cb8xxaUWazlcM9ceOlo0InDubL+J8zdg=="
+          files:
+            - .buildkite/secrets/tarball_signing.gpg
+            - .buildkite/secrets/macos_codesigning.keychain.encrypted
+    timeout_in_minutes: ${TIMEOUT?}
```

```suggestion
    timeout_in_minutes: 90
```

Let's see if this fixes CI, and then we can figure out the interpolation later.